### PR TITLE
Add Documentation Using Docsify

### PR DESCRIPTION
**Pull request by therynamo**
I figured we could use a more parseable documentation format than github `README`s. 

You can test by pulling down the branch and running `docsify serve ./docs --port=3001` (given you have docsify globally installed).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,179 +1,10 @@
...
-```
-
-After you've written to the `.drone.yml`, you can run the following in the directory where you've saved your file:
-
-`GITHUB_TOKEN=<your_token> SLACK_URL=<your_webhook_url> drone exec`
-
-(Drone will load the environment variables you pass to it, that way you can simulate secrets with the `exec` command).
-
-This process replicates what the the plugin goes through when running live. If the run passes and you see what you expect, there will be a much higher confidence that things are operating correctly.
+[See More On Our Captain's Log Site]()
```
Comment (AndersDJohnson) [inline README.md]: Should this have an actual URL?

File: README.md [hunk 2]
```diff
@@ -1,179 +1,10 @@
-# Captains Log
+# Captain's Log
```
Comment (AndersDJohnson) [inline README.md]: I was wondering if/when this would get an apostrophe...

File: README.md (hunk 1 above)
Comment (therynamo) [reply README.md]: Yes, need to publish the github pages before I can add it 😄

File: README.md (hunk 2 above)
Comment (therynamo) [reply README.md]: Yeah, I was behind the ball on this one.

**Pull request merged**